src/components/Manifest.js

```diff
@@ -49,7 +42,7 @@ const StandardManifest = props => {
     pageSizes = DEFAULT_PAGE_SIZES,
     pageSizeLabelGenerator = DEFAULT_PAGESIZE_LABEL_GENERATOR,
     statusMessageGenerator = DEFAULT_STATUS_MESSAGE_GENERATOR,
-    noResultsComponent,
+    NoResultsComponent = NoResultsHandler,
```

Comment:
WHy are you renaming this?

Reply:
Oh, sorry, I'm dumb.  I get it.

Reply:
lol, that is misleading, if it wasn't needed also in the lines above I might switch to using || in the JSX